devices/motorola-potter/kernel/config.armv7l

```diff
@@ -0,0 +1,4851 @@
+#
```

Comment:
Delete this file, unless you have validated with armv7l...

(I probably should for addison...)